### PR TITLE
Feature/fix issue 1364

### PR DESCRIPTION
**Pull request by germanbisurgi**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| Is backward-compatible?    | ✔️
| Tests pass?   | ✔️
| Fixed issues  | #1364
| Added CHANGELOG entry?   | ✔️

### CODE REVIEW

**Review by schmunk42 — APPROVED** [review]

**Comment by YuriHeupa** [comment]
For some reason after this change was introduced, the disable_array_delete_all_rows/disable_array_delete_last_row options are no longer working as expected. When the editor loads a schema with empty initial data for an array, the "delete all row" and "delete last row" buttons are always visible. Reverting this change seems to fix the issue. Additionally, adding and then removing a row also causes the buttons to disappear.

File: src/editors/object.js [hunk 1]
```diff
@@ -1109,6 +1109,7 @@ export class ObjectEditor extends AbstractEditor {
 
     Object.keys(this.editors).forEach(i => {
       if (this.editors[i].isActive()) {
+        this.editors[i].refreshValue()
```
Comment (YuriHeupa) [inline src/editors/object.js:1112]: For some reason after this change was introduced, the disable_array_delete_all_rows/disable_array_delete_last_row options are no longer working as expected. When the editor loads a schema with empty initial data for an array, the "delete all row" and "delete last row" buttons are always visible. Reverting this change seems to fix the issue. Additionally, adding and then removing a row also causes the buttons to disappear.

Comment (schmunk42) [reply src/editors/object.js:1112]: Could you create an issue with an example.

Comment (YuriHeupa) [reply src/editors/object.js:1112]: Sure: https://github.com/json-editor/json-editor/issues/1396

**Pull request merged**